10-htcondor.conf

```diff
@@ -0,0 +1,4 @@
+[condor_master]
+command=/sbin/condor_master -f
```

Comment:
Minor, but officially the path is `/usr/sbin/condor_master` in the rpm.  I know rhel makes them the same thing with symlinks but i have mixed ideological feelings about relying on that..  (`/bin/entrypoint.sh` is another story since it is put there explicitly in the Dockerfile.)

Reply:
Good call! I had copied this from elsewhere